src/itest/kotlin/nl/lifely/zac/itest/client/ItestHttpClient.kt

```diff
@@ -161,6 +168,8 @@ class ItestHttpClient {
             .add("Content-Type", "application/json")
             .build()
 
-    private fun cloneHeadersWithAuthorization(headers: Headers): Headers =
-        headers.newBuilder().add("Authorization", "Bearer ${KeycloakClient.requestAccessToken()}").build()
+    private fun cloneHeadersWithAuthorization(headers: Headers, openZaakAuth: Boolean = false): Headers {
```

Comment:
Please start the name of a boolean with a verb, ie. `isOpenZaakAuth`, `useOpenZaakAuth`, or similar, and prefer `is..` in most cases!

Reply:
Fixed! Thanks!